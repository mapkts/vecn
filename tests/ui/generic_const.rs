#[vecn::vector]
struct TupleStruct<const T: usize>(T);

#[vecn::vector]
struct NamedStruct<const T: usize> {
    x: T,
}

fn main() {}
