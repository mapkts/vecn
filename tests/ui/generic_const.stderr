error: expected generic type, but found generic const
 --> $DIR/generic_const.rs:2:20
  |
2 | struct TupleStruct<const T: usize>(T);
  |                    ^^^^^^^^^^^^^^

error: expected generic type, but found generic const
 --> $DIR/generic_const.rs:5:20
  |
5 | struct NamedStruct<const T: usize> {
  |                    ^^^^^^^^^^^^^^
